processor.py

```diff
@@ -117,3 +123,37 @@ def pybasic_illumination_correction(self, brightfield_images: np.ndarray):
             corrected_movie = brightfield_images_corrected.astype(np.uint8)
 
             return corrected_movie
+
+def process_movies(fiji_path, movie_dir, frames, down_factor):
+    fiji = BasicpyPreprocessor(fiji_path)
+    for plate_dir in movie_dir.iterdir():
+        for well_dir in plate_dir.iterdir():
+            for movie_path in well_dir.iterdir():
+                movie = fiji.load_mitocheck_movie_data(movie_path)
+                corrected_movie = fiji.pybasic_illumination_correction(movie)
+                # change to control image compression
+                down_factor = 20
```

Comment:
i like moving this to a function, but consider adding these constants as default function args